### DISCUSSION TITLE
How do you uninstall TagStudio

### DISCUSSION THREAD

**Discussion by AlTimofeyev** [TagStudioDev/TagStudio#234] [Q&A] [answered]
## How Do You Uninstall TagStudio

I noticed that the Uninstall section was missing from the README file and decided to come here to ask about it. This sort of thing might be dumb to ask because it's supposed to be intuitive, but I just wanted to double check and ask here.

I'm running on an intel-based MacOS and I'm used to downloading `.dmg` files for installing an application and to uninstall them I just move the application to the trash bin. However, when downloading an application from source, like with this project, how would I go about uninstalling it? Do I have to manually delete files/folders or is there an automated script I can run from CLI?

I'm asking all of this before I install the application to test it out; I just saw the YT video that was posted.

**Comment by CyanVoxel** [comment, marked as answer]
The zipped downloads for Mac in the [releases](https://github.com/TagStudioDev/TagStudio/releases) section contain a `.app` file like you would usually find inside a `.dmg` - the application is completely self contained outside of a `.plist` file stored inside of `~/Library/Preferences/`.

By "downloading an application from source" do you mean building it from the source code? Either way the project doesn't "install" itself outside of the source folder or executable, minus the mentioned preference file and any `.TagStudio` library folders generated when you create libraries.

**Reply by AlTimofeyev** [reply]
This answered my question! THANK YOU!!!!

Yes! When I said "downloading an application from source" I did mean building it from source code. I'm used to having to build projects/applications when I download them from GitHub, so I just wanted to see if this application would also need to be built from source, and if so, then how would I go about uninstalling it later on. But you cleared up my question, Thank you!

Now I know I can just drag the `.app` file to the trash to get it uninstalled later on (aside from the `.plist` and `.TagStudio` things you mentioned). 

Looking forward to testing out your app! c:
